tests/gui/models/drive-constraints.spec.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+const m = require('mochainon');
+const angular = require('angular');
+require('angular-mocks');
+
+describe('Browser: SelectionState', function() {
+
+  beforeEach(angular.mock.module(
+    require('../../../lib/gui/models/drive-constraints')
+  ));
+
+  describe('DriveConstraintsModel', function() {
+
+    let DriveConstraintsModel;
+
+    beforeEach(angular.mock.inject(function(_DriveConstraintsModel_) {
+      DriveConstraintsModel = _DriveConstraintsModel_;
+    }));
+
+    it('should be an instance of `lib.src.DriveConstraints`', function() {
```

Comment:
Any reasons for the periods though? Referencing the actual file path (`lib/src/drive-constraints.js`) looks cleaner IMHO.